guibot/target.py

```diff
@@ -388,42 +372,41 @@ def __init__(self, value=None, text_filename=None, match_settings=None):
             # text generated on the fly is also acceptable
             pass
 
-    def __str__(self):
+    def __str__(self) -> str | None:
```

Comment:
I expect we should always have a string to print, even an empty one.

Reply:
Done